api/routes/authRouter/index.js

```diff
@@ -0,0 +1,71 @@
+const express = require("express");
+const DB = require("../../../data/helpers/auth");
+const router = express.Router();
+const bcrypt = require("bcryptjs");
+const jwt = require("jsonwebtoken");
+require("dotenv").config();
+
+// function helpers
+function generateToken(user) {
+  user = Object.assign({}, user, { password: "******" });
```

Comment:
You could also do it another way with spread syntax
```suggestion
  user = { ...user,  { password: "******" } };
```

Same either way 👍